src/main/java/org/dcsa/bkg/model/transferobjects/ChargeTO.java

```diff
@@ -0,0 +1,38 @@
+package org.dcsa.bkg.model.transferobjects;
+
+import lombok.Data;
+import org.dcsa.bkg.model.enums.TransportPlanStage;
+import org.dcsa.core.events.model.Location;
+import org.dcsa.core.events.model.ModeOfTransport;
+import org.dcsa.core.events.model.enums.PaymentTerm;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@Data
+public class ChargeTO {
+
+  @NotNull(message = "Charge Type is required.")
+  @Size(max = 20, message = "Charge Type has a max size of 20.")
+  private String chargeType;
+
+  @NotNull(message = "Currency Amount is required.")
+  private Number currencyAmount;
```

Comment:
Use a concrete implementation instead of `Number` since it should in a way align with the IM too.